### DISCUSSION TITLE
TaskEditDialog Localization for chinese

### DISCUSSION THREAD

**Discussion by shengchungshih** [dahall/TaskScheduler#932] [Q&A] [answered]
I want to show TaskEditDialog with Chinese (Traditional), how can I configure or how can I Merge localized resources to Chinese (Traditional) ?

**Comment by dahall** [comment]
Sorry for my ignorance, but which locale code is Chinese (Traditional)? The library already supports "zh-CN". To use that, you just need to set the culture of your application to that locale. If you need new translations for localizing to Chinese (Traditional), then I can provide a spreadsheet for you to complete and then I'll incorporate it.

**Reply by shengchungshih** [reply]
Thanks for your reply , please send spreadsheet  to mail:  prophet1114@gmail.com  ,  I will translate to Chinese (Traditional).

**Reply by dahall** [reply]
Which specific localization do you want, "zh-Hant" or something else? Once you tell, me I'll send the spreadsheet.  Would it help to include the English and/or the Simplified Chinese in the spreadsheet?

**Comment by dahall** [comment, marked as answer]
@shengchungshih Below is the spreadsheet. If "zh-Hant" is not the locale you want, feel free to change the heading of that column to the local you need. You can see all the languages, but the English text is locked into view and the Simplified Chinese is in the adjacent column. Thank you for your offer to translate.
[TaskSchedLocalization2022_01_31.xlsx](https://github.com/dahall/TaskScheduler/files/8008465/TaskSchedLocalization2022_01_31.xlsx)

**Reply by shengchungshih** [reply]
Thanks a lot, I will translate to "zh-Hant".

**Reply by shengchungshih** [reply]
@dahall  Here is the "zh-Hant" spreadsheet 
[TaskSchedLocalization2022_02_09_final.xlsx](https://github.com/dahall/TaskScheduler/files/8032380/TaskSchedLocalization2022_02_09_final.xlsx)

**Reply by dahall** [reply]
Thank you. I'm incorporating it now and will post an updated release shortly.

**Reply by dahall** [reply]
Should be visible shortly in NuGet as 2.10.1.
